src/documents/tutorial-developerIntroduction/DeveloperIntroductionToInfusionFramework-OpeningRemarks.md

```diff
@@ -0,0 +1,598 @@
+ * [`fluid.modelComponent`](/infusion/development/ComponentConfigurationOptions.md#model-components), which adds support for mutable state, model events, and state coordination and relay between different components.
+ * [`fluid.viewComponent`](/infusion/development/ComponentConfigurationOptions.md#view-components), which supplement model components with functionality to bind to the DOM of a web page.
+
+Component-based development emphasizes declarative configuration, loose coupling and flexible hierarchies. The framework manages component lifecycle and scoping and provides many supports to allow components to work in collaboration with one another.
+
+A new Infusion component is defined using the `fluid.defaults` function and a very basic "Hello, World!" component might look like this:
+
+<div class="infusion-docs-note">You can check out the [Live Example](http://codepen.io/waharnum/pen/oBYvwx?editors=1111) of the code below on [CodePen](http://codepen.io/)</div>
+
+
+```
+// The first argument is the name of the new component, the second is the
+// new component's default configuration
+fluid.defaults("fluid.helloWorld", {
+    // gradeNames supplies an array of strings that name previously
+    // defined Infusion components
+    gradeNames: ["fluid.component"]
+});
+```
+
+After definition, instances of the component can be created by calling the component's name as a function, with the option to pass in further configuration options as an argument:
+
+```
+var helloWorld = fluid.helloWorld();
+```
+
+Right now this component doesn't do anything new, but we will evolve its definition throughout this introduction to demonstrate further core concepts of the framework.
+
+### Invokers
+
+All Infusion components support the definition of public functions using [invokers](/infusion/development/Invokers.md); among other things, invokers allow related components to have different internal implementations of a function but present a common function to be called by other code. If we imagine a "Hello, World!" component, there are a variety of ways it might say "hello". Two of them are:
+
+* By printing to the web developer console via `console.log`
+* By displaying a message on a web page
+
+We'll define the first approach as an invoker on the "Hello, World!" component. Other styles of invokers are possible, but we'll use one here that lets us refer to functions of existing Javascript objects:
+
+<div class="infusion-docs-note">You can check out the [Live Example](http://codepen.io/waharnum/pen/MJbgEx?editors=1111) of the code below on [CodePen](http://codepen.io/)</div>
+
+```
+fluid.defaults("fluid.helloWorld", {
+    gradeNames: ["fluid.component"],
+    invokers: {
+        sayHello: {
+            // Configures this invoker to use the console object
+            "this": "console",
+            // Configures this invoker to use the log function of the console
```

Comment:
An existing phrasing I can find for this concept (in `C#`) is "contextual keywords": https://msdn.microsoft.com/en-us/library/x53a06bb.aspx#Anchor_0, which I think would be good enough for having a reasonable precise term to describe that a key such as `args` will be taken by the framework to have a particular meaning (and corresponding action taken by the framework to wire behaviours together) in certain contexts when encountered in parsing a configuration.

The Wikipedia entry (https://en.wikipedia.org/wiki/Reserved_word) notes the following:

> "A closely related and often conflated notion {to a reserved word} is a *keyword* which is a word with special meaning in a particular context."